apps/molgenis-components/src/components/forms/InputHyperlink.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <FormGroup
+    :id="id"
+    :label="label"
+    :required="required"
+    :description="description"
+    :errorMessage="stringError"
+  >
+    <InputGroup>
+      <template v-slot:prepend>
+        <slot name="prepend"></slot>
+      </template>
```

Comment:
@mswertz I copied this from the InputEmail, but are these needed here (and for email?) or can they be omitted?

Reply:
this was meant if we would like to include an icon. Probably we can do this better in the future.